### DISCUSSION TITLE
Connection timed out issue during upgrade to 0.64.3

### DISCUSSION THREAD

**Discussion by dheeban-k** [outline/outline#3667] [Self Hosting]
While we try to upgrade from 0.63.0 to 0.64.3 I followed these steps:
Referece: https://hub.docker.com/r/outlinewiki/outline
1. Pulled the latest image "docker pull outlinewiki/outline"
2. Had my .env file with appropriate values. Since i'm just upgrading the version I didn't touch any values in the .env file
3. Now ran the cmd, "docker run --env-file=.env outlinewiki/outline"
4. On running this i'm getting the error 

yarn run v1.22.18
$ node ./build/server/index.js
{"label":"lifecycle","level":"info","message":"\nIs your team enjoying Outline? Consider supporting future development by sponsoring the project:\n\nhttps://github.com/sponsors/outline\n"}
{"level":"warn","message":"Retrying redis connection: attempt 1"}
{"level":"warn","message":"Retrying redis connection: attempt 2"}
{"level":"warn","message":"Retrying redis connection: attempt 3"}
{"level":"warn","message":"Retrying redis connection: attempt 4"}
{"level":"warn","message":"Retrying redis connection: attempt 5"}
{"level":"warn","message":"Retrying redis connection: attempt 6"}
{"level":"warn","message":"Retrying redis connection: attempt 7"}
{"level":"warn","message":"Retrying redis connection: attempt 8"}
{"level":"warn","message":"Retrying redis connection: attempt 9"}
{"level":"warn","message":"Retrying redis connection: attempt 10"}
{"level":"warn","message":"Retrying redis connection: attempt 11"}
{"level":"warn","message":"Retrying redis connection: attempt 12"}
/opt/outline/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:186
                reject(new sequelizeErrors.ConnectionError(err));
                       ^

ConnectionError [SequelizeConnectionError]: connect ETIMEDOUT 10.174.48.8:5432
    at Client._connectionCallback (/opt/outline/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:186:24)
    at Client._handleErrorWhileConnecting (/opt/outline/node_modules/pg/lib/client.js:305:19)
    at Client._handleErrorEvent (/opt/outline/node_modules/pg/lib/client.js:315:19)
    at Connection.emit (node:events:526:28)
    at Connection.emit (node:domain:475:12)
    at Socket.reportStreamError (/opt/outline/node_modules/pg/lib/connection.js:53:12)
    at Socket.emit (node:events:526:28)
    at Socket.emit (node:domain:475:12)
    at emitErrorNT (node:internal/streams/destroy:157:8)
    at emitErrorCloseNT (node:internal/streams/destroy:122:3)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  parent: Error: connect ETIMEDOUT 10.174.48.8:5432
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
    errno: -110,
    code: 'ETIMEDOUT',
    syscall: 'connect',
    address: '10.174.48.8',
    port: 5432
  },
  original: Error: connect ETIMEDOUT 10.174.48.8:5432
      at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1157:16) {
    errno: -110,
    code: 'ETIMEDOUT',
    syscall: 'connect',
    address: '10.174.48.8',
    port: 5432
  }
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command."

5. I double checked my Redis IP and PostgreSQL Ip and the network settings as well. No luck

can u please help me with this @tommoor ?

**Comment by tommoor** [comment]
Just seems like redis isn't running to be honest 🤷‍♂️

**Comment by dheeban-k** [comment]
But how does my production is running !?
That's the confusing part

**Comment by dheeban-k** [comment]
This issue is fixed. It was my bad, i tried to reach redis via an unauthorised network. Now the error I'm facing after upgrading to 0.64.3v is I can export my data but on trying to import it throws 524 error after few minutes @tommoor

**Comment by dheeban-k** [comment]
Whereas, Import export is still successful in the verison 0.63.0
can u please help me on this? @tommoor
